### ISSUE TITLE
F_X

### ISSUE THREAD

**Issue by liuxin-SH** [zlwang7/S-FAST_LIO#10]
您好，請問您的F_X中是不是缺少了EXP(-w^ *t )。
![image](https://github.com/zlwang7/S-FAST_LIO/assets/56961236/1ed5056b-3ac0-4ad6-8122-7a778ee96e51)

**Comment by liuxin-SH** [comment]
也就是公式中第一行第一列。

**Comment by zhh2005757** [comment]
> 您好，請問您的F_X中是不是缺少了EXP(-w^ *t )。 ![image](https://private-user-images.githubusercontent.com/56961236/283084539-1ed5056b-3ac0-4ad6-8122-7a778ee96e51.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwNTM0NzQsIm5iZiI6MTcxNDA1MzE3NCwicGF0aCI6Ii81Njk2MTIzNi8yODMwODQ1MzktMWVkNTA1NmItM2FjMC00YWQ2LTgxMjItN2E3NzhlZTk2ZTUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI1VDEzNTI1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzYTczYzUzNjRlODYxY2NlM2ZiY2YwMWQ0NjQwZWI3MTc4NTczYzRhNzFmNWNkZGEwZjI0YjJiNDZkNTBkMDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.u2sBkgLl-jBIm2CoQA___QN4QZYkwfQXBoIOci6XAls)

原版其实就是单位阵，我理解由于dt比较小所以那个exp实际上近似于单位阵的，代码里直接用单位阵替代了

**Comment by narutojxl** [comment]
yes,  i have the same understanding,  https://github.com/hku-mars/IKFoM/issues/19
